fizzbuzz/src/test/java/fizzbuzz/FizzBuzzTest.java

```diff
@@ -9,7 +9,6 @@
 
 import static org.assertj.core.api.Assertions.*;
 
-@Disabled
```

Comment:
I like the idea of having them enabled, but I think we want users to remove this so tests pass for whole project fine